src/Umbraco.Web/Editors/UsersController.cs

```diff
@@ -557,6 +557,29 @@ public HttpResponseMessage PostEnableUsers([FromUri]int[] userIds)
 
             return Request.CreateNotificationSuccessResponse(
                 Services.TextService.Localize("speechBubbles/enableUserSuccess", new[] { users[0].Name }));            
+        }
+
+        /// <summary>
+        /// Unlocks the users with the given user ids
+        /// </summary>
+        /// <param name="userIds"></param>
+        public HttpResponseMessage PostUnlockUsers([FromUri]int[] userIds)
```

Comment:
awesome c# work @madsrasmussen  :) 

I'll need to update however because for locking/unlocking users we'll need to use ASP.NET Identity APIs. I'm still impressed though !